app/jobs/securities_import_job.rb

```diff
@@ -0,0 +1,10 @@
+class SecuritiesImportJob < ApplicationJob
+  queue_as :default
+
+  def perform(exchange_mic = nil)
```

Comment:
From a user perspective, my guess is that we're wanting to import by country code correct?

If that's the case, would `country_code = nil` be a better param to pass?  That way, we could do something like this in the job and make it 1 job per user:

```rb
def perform(country_code = nil)
  exchanges = StockExchange.in_country(country_code)
  market_stack_client = Provider::Marketstack.new(ENV["MARKETSTACK_API_KEY"])

  exchanges.each do |exchange|
    importer = Security::Importer.new(market_stack_client, exchange.mic)
    importer.import
  end
end
```

And then just a simple scope on `StockExchange`:

```rb
class StockExchange < ApplicationRecord
  scope :in_country, ->(country_code) { where(country_code: country_code) }
end
```

Reply:
Good call. Keeps us from having to arbitrarily map mic codes as well.